sycl/include/sycl/accessor.hpp

```diff
@@ -2673,6 +2669,11 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS accessor<
   // Use base classes constructors
   using local_acc::local_acc;
 
+public:
+  local_ptr<DataT> get_pointer() const {
+    return local_ptr<DataT>(local_acc::getQualifiedPtr());
+  }
```

Comment:
If we are defining the old `get_pointer` here, is there a point to keeping the one in `local_accessor_base` and not just moving it to `local_accessor`?

Reply:
thanks @steffenlarsen, the old get_pointer is removed.

Reply:
I understood your point. There was an internal debate about this after my first implementation, that there is no reason that **get_pointer** in **local_accessor** and **accessor** with **target::local** specialization should return different types as local_accessor and accessor with **target::local** specialization should be identical. That's why I removed that old implementation from target::local specialization. 
I can follow the specification and re-add the old implementation in target::local specialization, if you believe we should have it.

Reply:
I believe the old definition was kept for backwards compatibility and as such it is part of the standard, although deprecated. As such we should keep it around.

Reply:
@steffenlarsen , thanks, done.